### PR TITLE
Use js-beautify for formatting HTML

### PR DESCRIPTION
**Pull request by nitrohorse**
## Description

Related: https://github.com/privacytoolsIO/privacytools.io/pull/900

This PR introduces ["js-beautify"](https://github.com/beautify-web/js-beautify) for auto-formatting the HTML code on code commits and git pushes based on specified rules in `.jsbeautifyrc` ([here's a larger example](https://gist.github.com/wzup/fc3254562236c1ec3f69)). The rule I added for now includes using [tabs over spaces](https://old.reddit.com/r/javascript/comments/c8drjo/nobody_talks_about_the_real_reason_to_use_tabs/). Overall, this tooling will increase style consistency and maintainability for contributors :smile: 

Example of trying to commit unformatted code:

husky-fail

Example of committing unformatted code and Travis failing the build:

travis-checks-fail

travis-fail

Alternatively, @dawidpotocki mentioned [Prettier](https://prettier.io/) which could also do this job. I also have a working, local branch for that but felt Beautify to be more tailored and simpler for our needs at the moment. I also ran into some weird HTML formatting issues (such as https://github.com/prettier/prettier-vscode/issues/646) when using Prettier.

### CODE REVIEW

File: _includes/sections/email-providers.html [hunk 1]
```diff
@@ -5,7 +5,7 @@ <h1 id="email" class="anchor"><a href="#email"><i class="fas fa-link anchor-icon
 </div>
 
 <div class="table-responsive">
-	<table class="table sortable-theme-bootstrap" data-sortable>
+	<<<<<<< HEAD <table class="table sortable-theme-bootstrap" data-sortable>
```
Comment (JonahAragon) [inline _includes/sections/email-providers.html]: This doesn't seem right

Comment (nitrohorse) [reply _includes/sections/email-providers.html]: Good catch! Fixed.

File: pages/providers/vpn.html [hunk 2]
```diff
@@ -5,59 +5,60 @@
...
-    <p class="card-text text-danger">If you are looking for <strong>anonymity</strong>, you should use the Tor Browser <strong>instead</strong> of a VPN.</p>
-    <p class="card-text text-danger">If you're looking for added <strong>security</strong>, you should always ensure you're connecting to websites using HTTPS. A VPN is not a replacement for good security practices.</p>
-    <p class="card-text text-secondary">If you're looking for additional <strong>privacy</strong> from your ISP, on a public Wi-Fi network, or while torrenting files, a VPN may be the solution for you as long as you understand <a href="#info">the risks involved</a>.</p>
-    <a href="https://www.torproject.org/" class="btn btn-danger">Download Tor</a>
-    <a href="https://write.privacytools.io/my-thoughts-on-security/slicing-onions-part-1-myth-busting-tor" class="btn btn-outline-danger">Tor Myths &amp; FAQ</a>
-    <a href="#info" class="btn btn-outline-secondary">More Info</a>
-  </div>
-</div>
+<<<<<<< HEAD {% include sections/vpn.html %}=======<div class="card border-danger">
```
Comment (nitrohorse) [inline pages/providers/vpn.html]: What... hmm

Comment (nitrohorse) [reply pages/providers/vpn.html]: Okay, fixed.

**Comment by Mikaela** [comment]
This looks scarily big to me, but there are conflicting files so I won't read deeper right now.

**Comment by nitrohorse** [comment]
> This looks scarily big to me, but there are conflicting files so I won't read deeper right now.

Yeah, I'm unsure what direction we want to take with this at the moment. It should be fairly straightforward to get auto-formatting tooling integrated, but for the proposed solutions (Prettier and JS-Beautify) both will introduce NodeJS to the project which will add some complexity and increase maintainability a bit (making sure packages are up-to-date)...

**Comment by nitrohorse** [comment]
Going to close this until the right formatter tool is decided :+1:

**Pull request closed without merging**